sdk/core/Azure.Core/src/Pipeline/RedirectPolicy.cs

```diff
@@ -15,12 +15,17 @@ namespace Azure.Core.Pipeline
     public sealed class RedirectPolicy : HttpPipelinePolicy
     {
         private readonly int _maxAutomaticRedirections;
-        private readonly bool _allowAutoRedirect = true;
+        private readonly bool _allowAutoRedirects = false;
 
-        internal static RedirectPolicy Shared { get; } = new RedirectPolicy();
+        internal static RedirectPolicy Shared { get; } = new RedirectPolicy(false);
 
-        private RedirectPolicy()
+        /// <summary>
+        /// Creates a new instance of the <see cref="RedirectPolicy"/> class.
+        /// </summary>
+        /// <param name="allowAutoRedirects">Determinds whether redirects will be handled by this policy. Rather than passing false, consider using the static <see cref="Shared"/> instance instead which defaults to false.</param>
+        internal RedirectPolicy(bool allowAutoRedirects)
```

Comment:
Nit: it's internal, so it totally doesn't matter, but this was singular previously because of the corresponding property name in HttpClient: https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.allowautoredirect?view=net-7.0